packages/atomic/package.json

```diff
@@ -86,24 +84,22 @@
     "@storybook/addon-essentials": "6.3.12",
     "@storybook/addon-viewport": "6.3.12",
     "@storybook/api": "6.3.12",
+    "@storybook/builder-webpack5": "^6.3.12",
```

Comment:
Is this needed to unify react version ?

If so, have you been able to `npm run storybook` and verify that it works okay ?

The storybook integration is quite custom !

Reply:
To unify the webpack version. Storybook still builds and runs fine 😄